src/components/info/info-panel.vue

```diff
@@ -2,18 +2,30 @@
 import { useTranslation } from 'i18next-vue'
 import SidePanelLayout from '@/components/common/side-panel-layout.vue'
 import { useAppStore } from '@/stores/app.store'
+import { useInfoStore } from '@/stores/info.store'
 import { storeToRefs } from 'pinia'
 import { useFeatureInfoStore } from '@/stores/feature-info.store'
+import useMap from '@/composables/map/map.composable'
+import LocationInfo from './location-info.vue'
 import FeatureInfo from '@/components/info/feature-info.vue'
-import { onUnmounted } from 'vue'
+import { watch, onUnmounted } from 'vue'
 
 const { t } = useTranslation()
 const appStore = useAppStore()
+const { locationInfo } = storeToRefs(useInfoStore())
+const map = useMap().olMap
+const { infoOpen } = storeToRefs(appStore)
 const { clearContent } = useFeatureInfoStore()
 const { featureInfoPanelContent, isLoading } = storeToRefs(
   useFeatureInfoStore()
 )
 
+watch(infoOpen, isOpen => {
+  if (!isOpen) {
+    clearContent()
+  }
+})
+
 onUnmounted(() => {
   clearContent()
 })
```

Comment:
If we really need the `v-show` above this component, we could get rid of the `onUnmounted` I guess ?

Reply:
you're right.

I could not set up the correct link to street view without the component being persistent